### ISSUE TITLE
Wrong numbers Italy March 12th

### ISSUE THREAD

**Issue by lcol90** [CSSEGISandData/COVID-19#611]

**Comment by lcol90** [comment]
THe number is 15113 and not 12462, source: https://www.worldometers.info/coronavirus/#countries

**Comment by maxdevblock** [comment]
Numbers for March 12th in Italy are

Confirmed: 15113
Deaths: 1016
Recovered: 1258

Source (Italian Departement of Health GitHub repository, updated everyday at 7:00-8:00 pm) https://github.com/pcm-dpc/COVID-19/blob/master/schede-riepilogative/regioni/dpc-covid19-ita-scheda-regioni-20200312.pdf

**Comment by lcol90** [comment]
Indeed, but your data say 12462 on the 12th of March :)

Extract from pandas

3/9/20      9172
3/10/20    10149
3/11/20    12462
3/12/20    12462

**Comment by maxdevblock** [comment]
> Indeed, but your data say 12462 on the 12th of March :) Extract from pandas 3/9/20 9172 3/10/20 10149 3/11/20 12462 3/12/20 12462

Yep, I know data here are wrong for Italy on March 12th (same of March 11th).
I was only adding Deaths and Recovered numbers too.
(I'm not a developer for this repository).

**Comment by lcol90** [comment]
ah ok :) thanks !

**Comment by JeanRintoul** [comment]
Also wrong numbers for UK, France and Australia. Looks like the last update for 12th of March has lots of data entry errors.

**Comment by maxdevblock** [comment]
On arcGIS too numbers are wrong. I guess they've got something broken in the main backend.
https://www.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6

**Comment by emilianbold** [comment]
Probably related to #650, all recent data is flawed.
